File: laravel_api/app/Models/Registry.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class Registry extends Model
{
    use HasFactory;

    protected $fillable = [
        'first_name',
        'last_name',
        'email',
        'user_id',
    ];

    public function user()
    {
        $this->belongsTo(User::class);
    }
}

File: menu_flutter/lib/models/app/registry.dart
class Registry {
  Registry({
    required this.id,
    required this.firstName,
    required this.lastName,
    required this.email,
  });

  int id;
  String firstName;
  String lastName;
  String email;

  static Registry fromJson(dynamic obj) {
    return Registry(
      id: obj['registry']['id'],
      firstName: obj['registry']['first_name'],
      lastName: obj['registry']['last_name'],
      email: obj['registry']['email'],
    );
  }
}
